app/models/school.rb

```diff
@@ -30,6 +30,7 @@ class School < ApplicationRecord
     '99' => 'SANS OBJET'
   }
   VALID_TYPE_PARAMS = %w[rep rep_plus qpv qpv_proche].freeze
+  SCHOOL_TYPES = %w[college lycee].freeze
```

Comment:
dans structure, on a des trucs bizarres : 
`    level character varying(100) DEFAULT 'lycee'::character varying NOT NULL,
    school_type public.school_category DEFAULT 'college'::public.school_category NOT NULL,`

Reply:
faut qu'on passe tout à lycee selon moi